flybywiresim/aircraft
# feat: Revised ECAM SD fuel page to match NEO

Fixes #[issue_no]

## Summary of Changes

The current fuel page is from the CEO variant. This revision is mostly cosmetic to correct the centre tanks to conform with the NEO.

## Screenshots (if necessary)



![Screenshot 2021-07-18 at 21 59 10](https://user-images.githubusercontent.com/5834254/137025530-cefbb68c-c99b-4fc9-99ff-5217acc88336.png)

![new-fuel-page](https://user-images.githubusercontent.com/5834254/137025557-a0abad0f-a56d-4a75-8441-97bccfe63a07.png)

## Reference





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Spawn in C&D, turn on batteries and set external power
2. Check centre tank pumps are amber until centre fuel pumps are turbed on using over head panel
3. Start APU and then engines
3. Fill centre tanks and partially deplete inner tanks
4. Observe fuel flow indication (dog-leg arrow poiting from centre tank pump to inner tank)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
